src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -221,5 +229,10 @@ private static bool IsInvalidGrantSubError(string subError)
 
             return !s_nonUiSubErrors.Contains(subError);
         }
+
+        private static string UpdateExceptionForClaimsChallenge(string message)
```

Comment:
Why is this method needed? This method has a single line operation which is used just once.